community-stabilization-fund-api/pages/donate/index.tsx

```diff
@@ -1,26 +1,28 @@
 import { Button } from 'carbon-components-react';
 
-import { BasicLayout } from '../../src/components';
+import { BasicLayout } from '../../src/components/BasicLayout';
+import { Footer } from "../../src/components/Footer";
 
 import styles from './donate.module.css';
 
 const Donate = () => (
-  <BasicLayout>
-    <div className={styles.donate}>
+  <div className={styles.donate}>
+    <BasicLayout>
       <p>
         If you would like to donate to the Pittsburgh Community Stabilization
         Fund, please follow the link below to Community Movement Builders, a
         founding partner non-profit organization for PCSF, and write ‘Pittsburgh
         Community Stabilization Fund’ or ‘PCSF’ in the comments section of the
         donation page and we will earmark those funds for this program.
       </p>
-      <a href='https://communitymovementbuilders.org/donate/'>
-        <Button kind='secondary' className={styles.btn_link}>
+      <a className={styles.btn_link} href='https://communitymovementbuilders.org/donate.html'>
+        <Button className={styles.btn_copy} kind='secondary'>
           CLICK HERE TO DONATE FOR PCSF AT COMMUNITY MOVEMENT BUILDERS
         </Button>
       </a>
-    </div>
-  </BasicLayout>
+    </BasicLayout>
+    <Footer />
+  </div>
```

Comment:
This is fine for now cause in the refactor we'll put the Footer and Header components all in a Layout component